rentalPrice.js

```diff
@@ -1,33 +1,27 @@
-// age - age of driver
-// licence - number of full years person holds driving licence
-// clazz - class of the car from 1 (smallest) to 5 (largest) that person wishes to rent
-// acc - has s/he caused any accidents within last year
-// acc2 - has s/he participated (but not caused) in any accidents within last year
-// season - if it is high season or not
-function price(age, licence, clazz, acc, acc2, season) {
+function price(driverAge, licenceAge, vehicleClass, accidentsCaused, accidentsParticipated, highSeason) {
 
-    if (age < 18)
+    if (driverAge < 18)
     {
       return "Driver too young - cannot quote the price";
     }
-    if (age <= 21 && clazz > 2)
+    if (driverAge <= 21 && vehicleClass > 2)
     {
         return "Drivers 21 y/o or less can only rent Class 1 vehicles";
     }
-    var rentalprice = age;
-    if (clazz >= 4 && age <= 25 && season !== false)
+    var rentalprice = driverAge;
+    if (vehicleClass >= 4 && driverAge <= 25 && highSeason !== false)
     {
       rentalprice = rentalprice * 2;
```

Comment:
If we follow camelCase logic, let's follow that everywhere. 
Additionally, does this follow business calculation correctly?